test/dRep_server_test.py

```diff
@@ -0,0 +1,340 @@
+    'SURF_B_2binners_CheckM_dRep': {
+        'genomes_refs': ['34837/17/13', '34837/18/13'] # maxbin, metabat
+        },
+    'capybaraGut_MaxBin2': {
+        'genomes_refs': ['37096/11/1'],
+        'bins_dir_name_l': ['capybaraGut.MaxBin2']
+        },
+    'capybaraGut_MetaBAT2': {
+        'genomes_refs': ['37096/9/1'],
+        'bins_dir_name_l': ['capybaraGut.MetaBAT2']
+        },
+    'capybaraGut_2binners': {
+        'genomes_refs': ['37096/11/1', '37096/9/1'],
+        'bins_dir_name_l': ['capybaraGut.MaxBin2', 'capybaraGut.MetaBAT2']
+        },
+    }
+
+param_sets = {
+
+    'numbers': {
+        'completeness_weight': 0.5,
+        'contamination_weight': 3.5,
+        'strain_heterogeneity_weight': 0.6,
+        'N50_weight': 0.111,
+        'size_weight': -0.3,
+
+        'MASH_sketch': 800,
+        'P_ani': 0.8,
+        'S_ani': 0.95,
+        'cov_thresh': 0.01,
+
+        'percent': 70,
+
+        'warn_dist': 0.255,
+        'warn_sim': 1.0,
+        'warn_aln': 0.22
+        },
+
+    'passive_options': {
+        'S_algorithm': 'ANIn',
+        'n_PRESET': 'tight',
+        'coverage_method': 'total',
+        'clusterAlg': 'ward',
+        'tax_method': 'max'
+        },
+
+    'filtering': {
+        'length': 2500000,
+        'completeness': 95,
+        'contamination': 5
+        },
+
+
+    'ignoreGenomeQuality': {
+        'ignoreGenomeQuality': 'True',
+        'completeness': 99,
+        },
+ 
+    'go_ANI': { # FAIL
+        'S_algorithm': 'goANI'
+        },
+
+
+   'ANImf_tight': { # normal may not apply to (default) ANImf
+        'S_algorithm': 'ANImf',
+        'n_PRESET': 'tight'
+        },
+
+   'ANIn_tight': {
+       'S_algorithm': 'ANIn',
+       'n_PRESET': 'tight'
+       },
+
+    'gANI_total': { # total does not apply to gANI
+        'S_algorithm': 'gANI',
+        'coverage_method': 'total',
+        },
+
+    'SkipMASH':  {
+        'SkipMASH': 'True',
+        'MASH_sketch': 900
+        },
+
+
+    'SkipSecondary': {
+        'SkipSecondary': 'True'
+        },
+
+    'SkipClustering': { # ?
+        'SkipMASH': 'True',
+        'SkipSecondary': 'True',
+        },
+
+    'SkipAll': { # ?
+        'ignoreGenomeQuality': 'True',
+        'SkipMASH': 'True',
+        'SkipSecondary': 'True',
+        }, 
+}
+
+#param_sets = {'filtering': param_sets['filtering']}
+#param_sets = {key: param_sets[key] for key in list(param_sets.keys())[:-2]}
+
+
+params_local = {
+#---------------- machine specific----------------------    
+    'processors': 8,
+    'checkM_method': 'taxonomy_wf', # default uses 40GB memory, this uses 16GB (?)
+#----------------- skip --------------------------------    
+    'skip_dl' : True,
+    #'skip_dRep': True,
+    'skip_save_bc': True,
+    'skip_save_shock': True,
+#----------------- test data for skips -----------------    
+    **file_combos['SURF_B_2binners_CheckM']
+}
+
+
+class dRepTest(unittest.TestCase):
+
+
+    # TODO if this throws, kill test suite
```

Comment:
Should this still be a TODO? Why not just make it the first test?

Reply:
i wrote this TODO because i would set off long test suites, and they would all come back to me at the same time at the end of the test with the same bug. i'd rather that if it can't past the most basic test, it just tells me before 2 hours have passed

Reply:
and it's still a TODO because i don't know how to implement it without doing a whole bunch of work